CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## unreleased
 
 * [CHANGE] TSDB: Fix the predicate checking for blocks which are beyond the retention period to include the ones right at the retention boundary. #9633
+* [FEATURE] SD: Support unregistering discovery manager metrics. #13896
```

Comment:
I didn't know that API changes are also added to the changelog.

Reply:
removed

Reply:
I'm not sure if we have clear guidelines for this but I'm inclined to add them all and then have the release manager have a final review on the changelog.